beacon-chain/main.go

```diff
@@ -98,6 +103,14 @@ func main() {
 			return fmt.Errorf("unknown log format %s", format)
 		}
 
+		//if the user has specified a log file name (--log-file arg) -
+		//we configure a persistent log file logger , a formatter and a hook.
+		logFileName := ctx.GlobalString(cmd.LogFileName.Name)
+		if logFileName != "" {
+			logFileFormatName := ctx.GlobalString(cmd.LogFileFormat.Name)
+			logutil.ConfigurePersistentLogging(logFileName, logFileFormatName)
```

Comment:
You must handle the error. If logutil returns some error, log it.

```suggestion
			if err := logutil.ConfigurePersistentLogging(logFileName, logFileFormatName); err != nil {
			    log.WithError(err).Error("Failed to configuring logging to disk.")
			}
```